### PR TITLE
Propagate `default_vhost` to zabbix::web

### PR DESCRIPTION
**Pull request by ikcalB**
#### Pull Request (PR) description
enables workaround for #825, fixes bug

#### This Pull Request (PR) fixes the following issues
parameter was not beeing propagated to zabbix::web, therefore rendering it
useless.

### CODE REVIEW

**Comment by ikcalB** [comment]
I don't think the PR change can cause errors on some tests, but succeed on most.

Skimping through the reasons why it failed, I guess the tests may need checking? Or maybe the master branch has some issues.
Maybe someone with more experience can chime in on this?

**Comment by ikcalB** [comment]
@pccibot I think the tests might fail because of something other than the PR. (one liner. if working on any version, should work on all)

can you please comment?

**Comment by smortex** [comment]
Recent PR have passing CI, I guess the error you see have been fixed :-). Can you please rebase your changes on top of the main branch so that we can sort this out?

From your working directory:

```
git fetch origin         # Download the latest code we have here
git rebase origin/master # Move your commits on top of the main branch
git push -f              # Send the changes (-f is required because we re-wrote history)
```

**Comment by ikcalB** [comment]
Hi! 
Will do that, thanks.

Am on vacation, will take 10 days, approx.

**Comment by ikcalB** [comment]
@smortex please have a look at the static validation checks AND the diff: I added a single line, in `init.pp@354` and static checks are complaining about opening whitespace error at `web.pp` (i.e. 410, and 4 more locations in that file)

TL;DR: static checks complain about upstream, not my change

**Comment by smortex** [comment]
Can you please rebase your changes on top of the main branch (CI should be fixed)?

From your working directory:

```
git fetch origin       # Download the latest code we have here
git rebase origin/main # Move your commits on top of the main branch
git push -f            # Send the changes (-f is required because we re-wrote history)
```

**Comment by ikcalB** [comment]
@smortex looks like the checks are passing now, thanks.

**Review by smortex — APPROVED** [review]
LGTM!
